Header header
string topic_name
uint64 bandwidth
uint64 frequency